frontend/assets/l10n/main/app_en.json

```diff
@@ -90,7 +94,18 @@
     "flashOff": "Flash off",
     "flashOn": "Flash on",
     "internetRequired": "An internet connection is required.",
+    "moreActionsActivateDescription": "Your saved Ehrenamtskarte will be retained. You can remove it manually.",
+    "moreActionsActivateLimitDescription": "To add another Ehrenamtskarte, you must first delete an existing Ehrenamtskarte.",
+    "moreActionsActivateTitle": "Add another Ehrenamtskarte",
+    "moreActionsApplyDescription": "Your saved card will be retained.",
+    "moreActionsApplyTitle": "Apply for or extend a Ehrenamtskarte",
```

Comment:
an
Even though Ehrenamtskarte is German, it starts with an E, so I would write "an" instead of "a"